docs/en/observability/user-experience.asciidoc

```diff
@@ -0,0 +1,185 @@
+and it only takes a few lines of code to <<instrument-apps,get started>>.
+
+The RUM agent uses browser timing APIs, like https://w3c.github.io/navigation-timing/[Navigation Timing],
+https://w3c.github.io/resource-timing/[Resource Timing], https://w3c.github.io/paint-timing/[Paint Timing],
+and https://w3c.github.io/user-timing/[User Timing], to capture {user-experience}
+metrics every time a user hits one of your pages.
+This data is stored in {es}, where it can be visualized using {kib}.
+
+[discrete]
+[[user-experience-tab]]
+== {user-experience} in {kib}
+
+// Need some kind of app introduction here
+// Not quite sure yet...
+// ...any ideas?
+
+[discrete]
+[[user-experience-page-load]]
+=== Page load duration
+
+How long is my server taking to respond to requests?
+How much time is spent parsing and painting that content?
+How many page views has my site received?
+This high-level overview is your analysis starting point.
+You won't be able to fix any problems from viewing these metrics alone,
+but you'll get a sense of the big picture as you dive deeper into your data.
+
+[role="screenshot"]
+image::images/page-load-duration.png[User experience page load duration metrics]
+
+[discrete]
+[[user-experience-metrics]]
+=== {user-experience} metrics
+
+{user-experience} metrics help you understand the perceived performance of your website.
+For example, first contentful paint is the timestamp when the browser begins rendering content.
+In other words, it's around this time that a user first gets feedback that the page is actually loading.
+
+[role="screenshot"]
+image::images/user-exp-metrics.png[User experience metrics]
+
+// This is collapsed by default
+[%collapsible]
+.Metric reference
+====
+First contentful paint::
+The timestamp when the browser begins rendering content (images, text, etc.) from the DOM.
+This is when the user can tell that the page is loading.
+footnote:[Source and more info: https://developer.mozilla.org/en-US/docs/Glossary/First_contentful_paint[developer.mozilla.org]]
+
+Total blocking time::
+The total amount of time that the user is blocked from providing input to the page
+-- no mouse clicks, button presses, etc.
+footnote:[Source and more info: https://web.dev/tbt/[web.dev]]
+
+Number of long tasks::
+Long tasks are periods of time where the main UI thread is working for 50ms or longer.
+This could be because of a rerender, event handler, etc., and causes problems like delayed time to interactivity
+and variable input latency.
+footnote:[Source and more info: https://web.dev/tbt/[developer.mozilla.org]]
+
+// Is this right?
+Longest long task duration::
+The longest of the long tasks.
+
+// Is this right?
+Total long tasks duration::
+Sum duration of all long tasks.
+====
+
+These metrics tell an important story about how users may judge your website.
+The problem is that developers shouldn't have to become experts at interpreting and acting on metrics like paint timings,
```

Comment:
May be we can rephrase this part a bit? We can call them as signals or opportunities to developers?

Reply:
I tried. Let me know what you think